docs/Installation.md

```diff
@@ -114,7 +114,11 @@ Trace linkage with APM is currently supported for following Splunk APM agents:
 
 If your configuration is unsupported, you can still stitch the APM trace with the RUM information by [enabling the trace linking manually](https://github.com/signalfx/splunk-otel-js-browser/blob/main/docs/ServerTraceContext.md).
 
+## Hybrid inline / body bottom mode
+
+If you decide not to load the instrumentation synchronously in the HEAD due to its size, but you would like to track of errors happening during the page load, you can add a separate small inline error tracker.
```

Comment:
A person reading this might ask, "How?  Where?", and in response we could provide more details here or a link to a specialized page outlining the complexities of this approach.